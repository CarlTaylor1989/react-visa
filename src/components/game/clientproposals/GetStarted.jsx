import React from 'react';
import { Translate } from 'react-redux-i18n';
import Arrows from '../../generic/arrows/Arrows';

import './GetStarted.scss';

const GetStarted = () => (
  <div className="getStarted">
    <span className="guide" />
    <Translate className="instructionText" value="clientprops.getStarted" />
    <Arrows />
  </div>
);

export default GetStarted;
