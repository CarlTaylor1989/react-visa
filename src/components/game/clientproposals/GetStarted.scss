@import '../../../styles/mixins';

$guide-icon: '../../../styles/icons/clients/guide_client_proposals.svg';
$guide-icon-width: 246px;
$guide-icon-height: 287px;

$arrows-down-icon: '../../../styles/icons/clients/arrows_down.svg';
$arrows-down-icon-width: 33px;
$arrows-down-icon-height: 31px;

.getStarted {
  @include align-items(center);
  display: flex;
  flex-direction: column;
  height: $guide-icon-height;
  justify-content: center;
  position: absolute;
  width: 1250px;

  .instructionText {
    font-size: 24px;
  }

  .guide,
  .arrowsDown,
  .divider {
    background-position: center center;
    background-repeat: no-repeat;
    display: block;
  }

  .guide {
    background-image: url($guide-icon);
    bottom: 0;
    height: $guide-icon-height;
    left: 100px;
    position: absolute;
    width: $guide-icon-width;
  }

  .arrows {
    margin-top: 25px;
  }
}
