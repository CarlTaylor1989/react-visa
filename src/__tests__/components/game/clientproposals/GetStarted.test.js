/* global toJson */
import React from 'react'; // eslint-disable-line no-unused-vars
import GetStarted from '../../../../components/game/clientproposals/GetStarted';

test('renders the GetStarted component', () => {
  const wrapper = shallow(<GetStarted />);
  expect(toJson(wrapper, { noKey: false })).toMatchSnapshot();
});
