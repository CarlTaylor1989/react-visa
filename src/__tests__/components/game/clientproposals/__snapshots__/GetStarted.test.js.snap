// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the GetStarted component 1`] = `
<div
  className="getStarted"
>
  <span
    className="guide"
  />
  <Translate
    className="instructionText"
    tag="span"
    value="clientprops.getStarted"
  />
  <Arrows
    direction="bottom"
  />
</div>
`;
